src/sentry/static/sentry/app/views/organizationIntegrations/integrationIcon.tsx

```diff
@@ -21,7 +21,16 @@ const Icon = styled('img')<IconProps>`
 
 const IntegrationIcon = ({integration, size}: Props) =>
   integration.icon ? (
-    <Icon size={size} src={integration.icon} />
+    <Icon
+      size={size}
+      src={integration.icon}
+      onError={e => {
+        (e.target as HTMLImageElement).src =
```

Comment:
@NisanthanNanthakumar I think we should avoid mutating the HTML `src` like this, it's not very React like. I think it would be better to create an error state for the component and provide a different value in `source` depending on whether there is an error r not.